### ISSUE TITLE
Error cloning stock fuel line

### ISSUE THREAD

**Issue by leatherneck6017** [DMagic1/KSP-EVA-Transfer#2]
Squad changed the case of the stock part folder from "fuelLine" to "FuelLine". This prevents the EVAFuelLine from being created. I updated it and it worked.

**Comment by DMagic1** [comment]
This is fixed in version 8.1: https://github.com/DMagic1/KSP-EVA-Transfer/releases/tag/8.1

But I'm going to leave this issue up since there will inevitably be people who have upgraded an existing KSP installation from KSP 1.3 to 1.4, and will then have the old directory name.

So if you are using version 8.0 and KSP 1.4.x and it works fine then continue to do so, otherwise upgrade to 8.1.

**Comment by gtfour1** [comment]
same story here. transfer fuel line just doesn`t appear in game, nowhere. 1.4.3, tried versions from7.1 to 8.1. also tried editing cfg. no success.

**Comment by DMagic1** [comment]
@gtfour1 

Does the spelling and capitalization of the EVA Transfer config file match the actual folder structure for the stock Fuel Line part? Squad changed the capitalization for 1.4, but that change won't actually happen if you are upgrading in place from KSP 1.3 or earlier. So you have to manually check that they match if things aren't working right.

**Comment by gtfour1** [comment]
yes, checked and doublechecked, after reading this thread. will check again, but have no idea why this keep happening. i mean, not happenning.

**Comment by DMagic1** [comment]
I need to see log files to say more. It sounds like either something is preventing the part from loading properly, or something is preventing it from being used when loaded. But I can't know which, or if it's something else without the log files.

**Comment by gordonfpanam** [comment]
I encountered this problem just today. KSP's log complained about cloning the stock transfer line's mesh because it couldn't find the mesh file.

The part file EVAFuelLine.cfg had a mesh path of "Squad/Parts/CompoundParts/FuelLine/model" with a capital F in "FuelLine." In KSP 1.4.3 this is a lower case F as in "fuelLine." Changing the case in EVAFuelLine.cfg corrected the problem, even though this was on Windows and generally file paths aren't case-sensitive here.

Oh, right: This KSP install was in-place upgraded from 1.4.1 to 1.4.3 via Steam, and from 1.3.1 prior to that. It probably kept the file path case from one of these earlier versions.
